### DISCUSSION TITLE
Can't access code-server

### DISCUSSION THREAD

**Discussion by king-anduin** [coder/code-server#2192] [Q&A]
Hi, 
### I installed code-server on my remote Ubuntu-Server 20. System is updated. Code-server is running without any issues. 
![Screenshot from 2020-10-11 22-26-10](https://user-images.githubusercontent.com/56738872/95689610-41ee1a00-0c12-11eb-973a-7b7c8e5f6968.png)

### That's my Apache configuration file:

![Screenshot from 2020-10-11 22-29-02](https://user-images.githubusercontent.com/56738872/95689617-487c9180-0c12-11eb-943a-54a34753a055.png)

### 
I'm connected via VPN server and I can't even get access to it if I try it locally. 
![Screenshot from 2020-10-11 22-30-14](https://user-images.githubusercontent.com/56738872/95689623-4ca8af00-0c12-11eb-8efa-e89887e2234f.png)

### I tried to change port I and IP of code-server but didn't work out.
I hope, you can help me out.

**Comment by code-asher** [comment]
Could you post your browser console log when you try to access
code-server via Apache?

I haven't ran code-server behind Apache before but maybe the discussion
here might help: https://github.com/cdr/code-server/discussions/2104

As for the VPN, I think if you access 127.0.0.1 you'd still be
connecting to your local computer. You'd need to use the local address
of the server, I think.
